DAS/simulator.py

```diff
@@ -63,15 +63,15 @@ def initValidators(self):
         assignedCols = []
         for i in range(self.shape.numberNodes):
             if self.config.evenLineDistribution:
-                if i < int(heavyVal/self.shape.vpn2):  # First start with the heavy nodes
-                    start =   i  *self.shape.chi*self.shape.vpn2
-                    end   = (i+1)*self.shape.chi*self.shape.vpn2
-                else:               # Then the solo stakers
-                    j = i - int(heavyVal/self.shape.vpn2)
-                    start = offset+(  j  *self.shape.chi)
-                    end   = offset+((j+1)*self.shape.chi)
-                r = set(rows[start:end])
-                c = set(columns[start:end])
+                if i < int(lightVal/self.shape.vpn1):  # First start with the light nodes
```

Comment:
Is there really a need to swap light and heavy nodes?

Reply:
The of who is class1 and who is class2 is currently duplicated in two files. These should be aligned, and swapping here is one way of doing that. My testing was all with local randomness, so I've seen this only when tried to refactor the changes into PRs